src/client.js

```diff
@@ -155,6 +155,13 @@ module.exports = class Client extends EventEmitter {
         // all cases we've been able to receive future notifications using the
         // same keys. So, we sliently drop this notification.
         this._persistentIds.push(object.persistentId);
+
+        // NOTE(ibash) for superhuman we want to keep an eye on these errors -
+        // so throw them in their own call stack so that they get caught by
+        // bugsnag.
+        setTimeout(() => {
+          throw error
+        }, 0)
         return
       } else {
         throw error
```

Comment:
Should we simplify this to:
```js
       if (error.message.includes('Unsupported state or unable to authenticate data')) {
         this._persistentIds.push(object.persistentId);
       }
       setTimeout(() => {
         throw error
       }, 0)
       return
```
Since we're throwing anyway…

Reply:
I think we want to throw the error synchrously when it's any other type. (Though, maybe it should emit error instead...)

Reply:
Ok, sounds good.